kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -247,29 +249,60 @@ define([
          * keys: files (list of file info) and error (optional error)
          */
         renderFiles: function (files) {
+            let stagingAreaViewer = this; 
```

Comment:
saving this because some of the inner functions were having trouble with the reference. wound up changing it everywhere in this function for consistency